content/en/continuous_integration/pipelines/gitlab.md

```diff
@@ -238,19 +238,20 @@ See the table below for the message and domain correlated with each error type.
 ## Enable job log collection
 
 The following GitLab versions support collecting job logs:
+
 * GitLab.com (SaaS)
-* GitLab >= 14.8 (self-hosted) only if using [object storage to store job logs][7]
+* GitLab >= 14.8 (self-hosted) only if you are using [object storage to store job logs][7]
 
 To enable collection of job logs:
 
-1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This reveals the `Enable logs collection` option in the Datadog integration.
-2. Enable the `Enable logs collection` option and save the changes.
+1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** check box on the [Pipeline Setup page][17].
```

Comment:
```suggestion
1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** checkbox on the [Pipeline Setup page][17].
```

Reply:
`checkbox` looks to be much more frequently used in the documentation site. This is also preferred in some style guides:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark